server-preload.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Use node.js preloading to require a node module before importing and executing
+ * the entrypoint script.
+ *
+ * Intended to be used in package.json scripts with something like:
+ * - `node -r ./server-preload.js ./node_modules/.bin/next start -p 8080`
+ *
+ * We need to connect to App Insights early enough for it to be able to log HTTP requests.
```

Comment:
Minor edit for clarity
```suggestion
 * We need to connect to App Insights early enough for it to be able to log HTTP requests,
 * so we use node.js preloading to require a node module before importing and executing
 * the entrypoint script.
 *
 * This script is called in package.json with:
 * - `node -r ./server-preload.js ./node_modules/.bin/next start -p 8080`
 *
```